interface/portraits/portraits_nightelf.gfx

```diff
@@ -11,36 +11,23 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
```

Comment:
These clothes are shifted to the right (you can see his bare shoulder UwU).
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/82448290-91dd1080-9aba-11ea-8cf4-2f9686892cc2.png)
![image](https://user-images.githubusercontent.com/46384992/82448313-97d2f180-9aba-11ea-899b-e9e74b7d98d5.png)
![image](https://user-images.githubusercontent.com/46384992/82448322-9e616900-9aba-11ea-8e3f-3831afcc76b4.png)
![image](https://user-images.githubusercontent.com/46384992/82448334-a3261d00-9aba-11ea-912a-87b1f9a171fd.png)
![image](https://user-images.githubusercontent.com/46384992/82448356-a9b49480-9aba-11ea-81f8-b8b87cdacc35.png)
![image](https://user-images.githubusercontent.com/46384992/82448372-b0dba280-9aba-11ea-823a-5ccf1ec14b40.png)

</details>